Sources/MapboxDirections/MapboxStreetClass.swift

```diff
@@ -0,0 +1,65 @@
+
+import Foundation
+
+public enum MapboxStreetClass: String, Codable {
```

Comment:
Can you also add conformance to `CaseIterable` protocol? Sometimes it might be useful when generating random street class.

Reply:
I can see how that would be helpful in some cases, but there isn’t an inherent ordering to all these classifications. For example, a motorway is larger than a trunk road, but is an aerialway larger than a service road?